submissions/RemoveBG_Extension/popup.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Image Clicked</title>
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      text-align: center;
+      padding: 20px;
+    }
+  </style>
+</head>
+<body>
+  <h1>Notification</h1>
```

Comment:
Over all I think the updates here are good. If you could change "Notification" to instead carry the name of the extension it'll be better. Also make text elements in there a bit bigger so it's easier on the eyes.